vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -91,9 +96,25 @@ func main() {
 	}
 
 	healthCheck := metrics.NewHealthCheck(time.Minute)
-	metrics.Initialize(*address, healthCheck)
 	metrics_admission.Register()
 
+	go func() {
+		pathRecorderMux := mux.NewPathRecorderMux("vertical-pod-autoscaler")
```

Comment:
reading the documentation and it looks like this name is used for logging, should we make this unique across each binary?

eg. here we would make it `vertical-pod-autoscaler-admission-controller`

Reply:
Fixed in e4fc6149e6